app/views/homepage/popular_links/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<h2 class="govuk-heading-m">Link <%= index +1 %></h2>
+<%= render "govuk_publishing_components/components/input", {
```

Comment:
We need to include the "With error items" option of the [Input component](https://components.publishing.service.gov.uk/component-guide/input#with_error_items) here so that the error is flagged on the element that has an error as well as the top of the page.